### PR TITLE
Changelog for 8.19

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -8,6 +8,510 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.19
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+TODO
```
Comment (SkySkimmer) [inline doc/sphinx/changes.rst:17]: (after the rc, not in this PR)

**Comment by SkySkimmer** [comment]
@coq/ssreflect-maintainers there were no changelog entries in the ssr section, is that correct?
https://github.com/coq/coq/pull/17920 and https://github.com/coq/coq/pull/18031 maybe should have changelogs?

**Review by jfehrle — APPROVED** [review]
LGTM

**Comment by SkySkimmer** [comment]
@jfehrle are you merging?

**Comment by jfehrle** [comment]
@coqbot: merge now

**Pull request merged**